src/_webrsp.m

```diff
@@ -70,6 +84,16 @@ D QSPLIT(HTTPREQ("query"),.HTTPARGS) I $G(HTTPERR) QUIT
  if "PUT,POST"[HTTPREQ("method") xecute "S LOCATION=$$"_r if 1
  else  do @r
  ;
+ ; if setting for json serialization is enabled then perform auto json serialization for response object
```

Comment:
I'm not sure the reason for this feature. The web server just sends the data back in the response variable. If you want it to be JSON data do it in your handling routine.

Reply:
Imagine 100's of routines returning json objects and we needing to duplicate the json serialization code in every routine. The server can easily handle such a scenario, most web-servers provide json serialization out of the box.